### ISSUE TITLE
Run on TPU

### ISSUE THREAD

**Issue by Pourbahman** [canjiali/PARADE#25]
Hi,

You told me "you should be able to run one fold within 12 hours on a tpu v2". 

Would you please tell me if I use bert base model each fold could be run within 12 hours on a tpu v. 2?

Also, would you please tell me at the end of each fold your implementation generates test data scores? If so, which fold has the best results? 

Thanks in advance,
Kind Regards

**Comment by canjiali** [comment]
hi, yes it's less than 12 hours for each fold. The scores are written in the output directory. the final result is the average of 5 folds.

**Comment by Pourbahman** [comment]
Thanks.
Excuse me, I mean **after running each fold** (not at the end of running all 5 fold) I have a ranked list like [https://zenodo.org/record/3974431/files/robust04.PARADE.runs.tar.gz](url) for test set?

**Comment by canjiali** [comment]
you can set a small number of training steps and see the output......

**Comment by Pourbahman** [comment]
You are right. I think num_train_steps in [https://github.com/canjiali/PARADE/blob/f330f12a0104c591d871aa54b3b0022dadaef512/run_reranking.py#L138](url)
would be what you said but would you please tell me where can I set it?

I think I should set it on [https://github.com/canjiali/PARADE/blob/master/scripts/run.reranking.sh](url) as a parameter but according to your comment "If this is set, the argument 'num_train_epochs' takes no effect". Would you please tell me if I set num_train_steps to, for example 5, as parameter of python3 -u run_reranking.py I can see the result as soon as possible?

```
 python3 -u run_reranking.py \
    --pretrained_model=electra \
    --do_train=True \
    --do_eval=True \
    --train_batch_size=32 \
    --eval_batch_size=32 \
    --learning_rate=$learning_rate \
    --num_train_epochs=$epoch \
    **--num_train_steps=5**
```

**Comment by canjiali** [comment]
yes
